Tests/conf.json

```diff
@@ -6217,7 +6217,13 @@
         "Aha": "No instance",
         "SlackV2": "Integration slackV2 is deprecated.",
         "DSPM": "No instance",
-        "Symantec Email Security Cloud": "No instance"
+        "Symantec Email Security Cloud": "No instance",
+        "RedLock": "Integration is deprecated.",
```

Comment:
If the integration is deprecated - remove the object from the conf.json - no need to skip it, it is not relevant anymore.